storage/src/http/objects/upload.rs

```diff
@@ -1,10 +1,67 @@
 use crate::http::object_access_controls::{PredefinedObjectAcl, Projection};
 
-use crate::http::objects::Encryption;
+use crate::http::objects::{Encryption, Object};
 use crate::http::Escape;
 use reqwest::header::{CONTENT_LENGTH, CONTENT_TYPE};
 use reqwest::{Client, RequestBuilder};
 
+pub struct Multipart {
+    pub boundary: String,
```

Comment:
Can you please document boundary as if defined by HTTP multipart.

Also it is preferable to use `Cow<'static, str>` for such parameters

Reply:
The implementation was changed drastically after it was found that it is possible to upload using only the multipart feature of `reqwest`.